python/mujoco/rollout.py

```diff
@@ -29,15 +29,14 @@ def rollout(model: mujoco.MjModel,
             *,  # require subsequent arguments to be named
             control_spec: int = mujoco.mjtState.mjSTATE_CTRL.value,
             skip_checks: bool = False,
-            nroll: Optional[int] = None,
             nstep: Optional[int] = None,
             initial_warmstart: Optional[npt.ArrayLike] = None,
             state: Optional[npt.ArrayLike] = None,
             sensordata: Optional[npt.ArrayLike] = None):
   """Rolls out open-loop trajectories from initial states, get subsequent states and sensor values.
 
   Python wrapper for rollout.cc, see documentation therein.
-  Infers nroll and nstep.
+  Infers nstep.
```

Comment:
Same as below, nroll is still inferred, just that now we are inferring something that is not exposed to Python

Reply:
Yes, for the same reason as the other comment.